curriculum/challenges/_meta/lecture-html-fundamentals/meta.json

```diff
@@ -0,0 +1,51 @@
+{
+  "name": "HTML Fundamentals",
+  "isUpcomingChange": true,
+  "dashedName": "lecture-html-fundamentals",
+  "order": 4,
+  "superBlock": "front-end-development",
+  "helpCategory": "HTML-CSS",
+  "blockType": "lecture",
+  "challengeOrder": [
+    {
+      "id": "670803abcb3e980233da4768",
+      "title": "What Are Divs and Spans, and When Should You Use Them?"
```

Comment:
I am not sure how I feel about casing for all of these titles.

Personally, I think we should go with only the first word starting with a capital letter and all other words as lowercase

Reply:
That's the structure we've always used for challenge titles. I think I am leaning towards keeping it like that - but could be convinced otherwise.

Reply:
If we have always used that structure then we can keep it the way it is